### PR TITLE
Add randomizeTargets action

### PR DESCRIPTION
**Pull request by alexhwoods**
You could argue this should be `shuffle`, but I think `random` is more phonetically unique

### CODE REVIEW

**Comment by alexhwoods** [comment]
Depends on https://github.com/pokey/cursorless-vscode/pull/440

**Review by pokey — COMMENTED** [review]
I think I prefer "shuffle" because it is a verb.  Did you experience accuracy issues with "shuffle"?

**Comment by alexhwoods** [comment]
Ok, just tested it out vs. `chuck` especially. It was fine, so I'm cool with that if you prefer

**Comment by pokey** [comment]
> Ok, just tested it out vs. `chuck` especially. It was fine, so I'm cool with that if you prefer

cool yeah "shuffle" would be my leaning then 👍

File: src/actions/actions_simple.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "pour": "editNewLineAfter",
     "pre": "setSelectionBefore",
     "puff": "insertEmptyLinesAround",
+    "random": "randomizeTargets",
```
Comment (alexhwoods) [inline src/actions/actions_simple.py:30]:
```suggestion
    "shuffle": "randomizeTargets",
```

**Pull request closed without merging**